imports/api/employee.test.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable no-undef */
+/* eslint-disable func-names */
+import { Meteor } from "meteor/meteor";
+import chai from "chai";
+import { Employees } from "./db";
+import "./comment";
+import "./employee";
+import "./task";
+import EMPLOYEEAPI from "../constant/methods/employeesAPI";
+
+
+const { expect } = chai;
+
+describe("Employee API Tests", function () {
+	Employees.remove({});
+	const _id = Meteor.call(EMPLOYEEAPI.INSERT, "firstname", "lastname");
```

Comment:
remember that the insert method takes more parameters. First of all you need to be signed in as admin, second there need to actually be a employee in the user account db. That's where we discovered that we can no longer rely on unit test since auth is always required. Here you can simply test if manually create a user and add it to the employee make the employee show up.